lib.js

```diff
@@ -2,6 +2,7 @@ const { Client, Guild, PermissionFlagsBits, PermissionsBitField, EmbedBuilder }
 const mysql = require('mysql2');
 const config = require('./config.json');
 const { generalErrorHandler } = require('./errorHandlers');
+const GoogleCalendar = require('GoogleCalendar');
```

Comment:
This should be `const GoogleCalendar = require('./GoogleCalendar.js');`

Without it will fail to import and throw `Error: Cannot find module 'GoogleCalendar'`.